src/grad.jl

```diff
@@ -1,40 +1,64 @@
 export grad, jacobian, jvp, j′vp, to_vec
+function replace_arg(x, xs::Tuple, k::Int)
+    return ntuple(length(xs)) do p
+        if p == k
+            x
+        else
+            xs[p]
+        end
+    end
+end
```

Comment:
why not put on one line as `ntuple(p -> p == k ? x : xs[p], length(xs))` ?

Reply:
yeah, sure. revised.